.github/workflows/actions.yml

```diff
@@ -190,7 +190,8 @@ jobs:
           sw_vers
       - name: Get Dependencies
         run: |
-          brew install cmake gettext libdeflate libevent libpsl miniupnpc ninja node pkg-config
+          brew uninstall -f pkg-config pkg-config@0.29.2
```

Comment:
unlink?

Reply:
unlink will fail the build when pkg-config is not installed... which happens randomly?